### PR TITLE
ninja: Update to 1.10.0

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

**Comment by Biswa96** [comment]
Do we need emacs as a build dependecy?

File: ninja/PKGBUILD [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
+pkgrel=1
 _descr="Ninja is a small build system with a focus on speed"
 pkgdesc="${_descr}"
 arch=('i686' 'x86_64')
 url="https://ninja-build.org"
 license=('Apache')
 depends=()
 options=('strip' 'staticlibs')
-makedepends=("re2c" "python3")
+makedepends=("re2c" "python3" "emacs")
```
Comment (lazka) [inline ninja/PKGBUILD]: why?

Comment (Biswa96) [reply ninja/PKGBUILD]: `package_ninja-emacs` is not built without it.

Comment (lazka) [reply ninja/PKGBUILD]: ah, right, thanks!

**Pull request merged**